core/templates/dev/head/editor/EditorServices.js

```diff
@@ -59,7 +59,17 @@ oppia.factory('explorationData', [
             $log.info('Retrieved exploration data.');
             $log.info(response.data);
 
-            explorationData.data = response.data;
+            // FIXME: Update this value based on what exactly is being sent from
+            // the backend (corresponding to the ‘current exploration’ data).
+            if (response.data.currentExplorationData) {
```

Comment:
Here I'm assuming that the response data from the backend will be in the specified format (as mentioned in the Editor Draft saving Backend doc). Again, as I mentioned above - it cannot be fixed till the backend implementation of this is done.